cocos/editor-support/cocostudio/CCSSceneReader.cpp

```diff
@@ -364,6 +364,10 @@ cocos2d::Node* SceneReader::createObject(CocoLoader *cocoLoader, stExpCocoNode *
     {
         className = pNodeArray[1].GetValue(cocoLoader);
     }
+    if (className == nullptr) {
```

Comment:
The assertion here is to catch before the strcmp, as the following code fragment contains:
```
    if(strcmp(className, "CCNode") == 0)
```

As far as I know, strcmp on a null value can potentially be dangerous, as can be seen in the following StackOverflow link:
http://stackoverflow.com/questions/4932848/strcmp-behaviour